README.md

```diff
@@ -7,7 +7,13 @@
 
 Get data from yaml file directly in your shell
 
+Niet is like sed for JSON/YAML data - you can use it to slice and filter and
```

Comment:
I do not agree with sed... sed can do really a lot of things and don't take care about languages, it manage only text .... 
 You could compare it to xmllint or jq (for json) but with the capacity to deal with yaml, json AND xml.
Here we can add/remove/edit som keys, but not so much compared to sed ...